modules/indexer/code/bleve.go

```diff
@@ -86,16 +91,23 @@ func openIndexer(path string, latestVersion int) (bleve.Index, error) {
 
 // RepoIndexerData data stored in the repo indexer
 type RepoIndexerData struct {
-	RepoID  int64
-	Content string
+	RepoID    int64
+	CommitID  string
+	Content   string
+	Language  string
+	UpdatedAt time.Time
 }
 
 // Type returns the document type, for bleve's mapping.Classifier interface.
 func (d *RepoIndexerData) Type() string {
 	return repoIndexerDocType
 }
 
-func addUpdate(update fileUpdate, repo *models.Repository, batch rupture.FlushingBatch) error {
+func addUpdate(commitSha string, update fileUpdate, repo *models.Repository, batch rupture.FlushingBatch) error {
+	// Ignore vendored files in code search
+	if enry.IsVendor(update.Filename) {
```

Comment:
This is being arbitrarily excluded?

Reply:
@guillep2k added option to disable vendored file exclusion from search